apis/Google.Cloud.Firestore.Data/Google.Cloud.Firestore.Data/ValueDeserializer.cs

```diff
@@ -0,0 +1,231 @@
+﻿// Copyright 2017, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Firestore.V1Beta1;
+using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Reflection;
+using static Google.Cloud.Firestore.V1Beta1.Value.ValueTypeOneofCase;
+using static Google.Cloud.Firestore.Data.SerializationHelpers;
```

Comment:
Should these be separated from the namespaces? I don't feel that strongly about it, but just wanted to make sure it wasn't unintentional VS auto-inserts.

Reply:
There are three namespace directives below the aliases.

Reply:
Not with the latest commit :)

Reply:
Ah...misread your comment (somehow skipped over the word "now").